src/main/java/muramasa/antimatter/capability/IInteractHandler.java

```diff
@@ -12,8 +12,8 @@
 
 @ParametersAreNonnullByDefault
 public interface IInteractHandler {
-
-    boolean onInteract(PlayerEntity player, Hand hand, Direction side, @Nullable AntimatterToolType type);
+    //ParsedSide represents the side as parsed by Utils.getInteractSide
+    boolean onInteract(PlayerEntity player, Hand hand, Direction side, @Nonnull Direction parsedSide, @Nullable AntimatterToolType type);
```

Comment:
Perhaps it should be called interactSide then?

Reply:
I am not sure, so have two separate interacts?
I might actually not need ParsedSide though, it is used for wire cutter & wrench for connecting machines.

Reply:
for now i will keep it but as needed split it